### PR TITLE
Update select "attribute" placeholders

### PR DESCRIPTION
**Pull request by Jesus89**
Related to #12498 

Add select placeholders:
- placeholder: "Select a column"
- disabled-placeholder: "Enable column"

For _label-attribute_ and _animated-attribute_

select-placeholder

### CODE REVIEW

File: lib/assets/core/locale/en.json [hunk 1]
```diff
@@ -2316,6 +2316,8 @@
         },
         "labels-attribute": {
           "label": "column",
+          "placeholder": "Select a column",
+          "disabled-placeholder": "Enable column",
```
Comment (nobuti) [inline lib/assets/core/locale/en.json]: I'm not sure of this. The disabled placeholder applies when the select component is part of the enabler editor component. In this scenario, even when the select was disabled, the user is not able of enabling it again. The checkbox only controls the block visibility. The select it's supposed to be enabled always right?

File: lib/assets/core/locale/en.json [hunk 2]
```diff
@@ -2347,6 +2349,8 @@
         },
         "animated-attribute": {
           "label": "column",
+          "placeholder": "Select a column",
+          "disabled-placeholder": "Enable column",
```
Comment (nobuti) [inline lib/assets/core/locale/en.json]: Same as above.

File: lib/assets/core/locale/en.json (hunk 1 above)
Comment (Jesus89) [inline lib/assets/core/locale/en.json]: Ok I see. "disabled-placeholder" is not used then.

File: lib/assets/core/locale/en.json (hunk 2 above)
Comment (Jesus89) [inline lib/assets/core/locale/en.json]: :+1:

**Comment by nobuti** [comment]
🚀

**Pull request merged**